packages/client-config/API.md

```diff
@@ -206,7 +207,10 @@ export type CustomClientConfig = {
 export const DEFAULT_CLIENT_CONFIG_VERSION: ClientConfigVersion;
 
 // @public
-export const generateClientConfig: <T extends "1" | "0">(credentialProvider: AwsCredentialIdentityProvider, backendIdentifier: DeployedBackendIdentifier, version: T) => Promise<ClientConfigVersionTemplateType<T>>;
+export const generateClientConfig: <T extends "0" | "1">(options: GenerateClientConfigOptions, backendIdentifier: DeployedBackendIdentifier, version: T) => Promise<ClientConfigVersionTemplateType<T>>;
+
+// @public (undocumented)
+export type GenerateClientConfigOptions = GraphqlModelsFetchOptions;

```

Comment:
I'd avoid reusing type from different package here. it couples their public API.

Reply:
👍  I wasn't sure if it was preferable to reuse to redefine a duplicate. I'll update it to duplicate the type definition.